src/components/Button.js

```diff
@@ -41,15 +43,13 @@ module.exports = React.createClass({
 		};
 	},
 	render() {
+		var classes = this.props.sheet.classes;
```

Comment:
Also I am interested to know how selector engine gets slower compared to growing css size.